fe/fe-core/src/main/java/org/apache/doris/qe/QeProcessorImpl.java

```diff
@@ -295,6 +318,18 @@ public String getSql() {
         public long getStartExecTime() {
             return startExecTime;
         }
+
+        public void setQueueToken(QueueToken queueToken) {
```

Comment:
remove setQueueToken and  getQueueToken setStartExecTime
Get these info from coordinator directly.